src/components/ListItem.vue

```diff
@@ -7,22 +7,21 @@
             <v-text-field
               style="margin-bottom:20px;"
               :value="listItem.text"
-              @change="listItem.text = $event"
+              @change="updateItem($event)"
```

Comment:
It turns out this may not be the correct event to use. Can you investigate the code for v-text-field and determine which event this binds to? Is that event the correct one? Read more about the events available on text inputs and determine which one you need to use, then see if the v-text-field responds to that event. 
NOTE: Mozilla developer network (aka MDN) is the definitive resource for JS, HTML, and CSS. If you want more than what is there, you should read the W3C specifications.